src/scripts/BackgroundScript.js

```diff
@@ -78,10 +77,10 @@ export default function getBackgroundScript(location) {
 
 					var deferredResponse = $.Deferred();
 
-					browser.runtime.sendMessage(reqToBackgroundScript).then(function(response) {
+					browser.runtime.sendMessage(reqToBackgroundScript).then(function (response) {
```

Comment:
In then we could use standard arrow functions: `response => deferredResponse.resolve(response)`